stories/atoms/components/toast/Toast.stories.mdx

```diff
@@ -316,6 +321,62 @@ These two elements are optional and are not inter-related.
               severityLevel="success"
             />
           </div>
+          <div>
+            <Button
+              label="Bot Anik"
+              onClick={handleState('cool', true)}
+              backgroundColor="primary"
+            />
+            <Toast
+              title="Bot Anik"
+              description={
+                <div style={{ display: 'flex', flexDirection: 'column' }}>
+                  <Typography
+                    as="span"
+                    color="highlighted-text"
+                    fontSize="small"
+                    fontWeight="light"
+                  >
+                    Bot Anik is a bot like no other, she'll kick you around without any trouble! She
+                    gets rid of all who messes around, no one can beat our bot! 🎵
+                  </Typography>
+                  <Typography
```

Comment:
If you don't mind, I'd prefer the use of the `Button` component instead of a custom element